index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Home</title>
+</head>
+
+<body>
+    <section class="home-section">
+        <header class="header1">
+            <div class="header1-content">
+                <ul>
+                   <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+                </ul>
+            </div>
+        </header>
+        <header>
+            <img src="img/menu.png" width="40" height="40" alt="hamburger-button" class="hamburger-button">
+            <img src="img/close.png" width="40" height="40" alt="close-btn" class="close-btn">
+            <nav class="navbar" id="navbar-list">
+                <ul>
+                    <li id="logo"><a href="index.html" rel="noreferrer noopener"><img src="img/logo.jpg" alt="logoimage" width="130" height="50"></a></li>
+                    <li><a href="about.html" rel="noreferrer noopener">About</a></li>
+                    <li><a href="#" rel="noreferrer noopener">Program</a></li>
+                    <li><a href="#" rel="noreferrer noopener">Join</a></li>
+                    <li><a href="#" rel="noreferrer noopener">Sponsor</a></li>
+                    <li><a href="#" rel="noreferrer noopener">News</a></li>
+                    <li id="websummit"><a href="#" rel="noreferrer noopener">Web Summit 2021</a></li>
+                </ul>
+            </nav>
+        </header>
+        <div class="main-title">
+            <h2>"Hello,Tech World"</h2>
+            <h3>Technology Conference<br>Web Summit<br>2021</h3>
+        </div>
+        <div class="description">
+            <p>Web Summit brings together the people and companies redefining the global tech industry.
+                Web Summit will return to Lisbon this year.</p>
+        </div>
+        <div class="date-time">
+            <h2>November 1-4, 2021</h2>
+            <h3>@ ALTICE ARENA & FIL, LISBON</h3>
+        </div>
+        <div class="startup-program">
+            <h2>Startup Program</h2>
```

Comment:
The changes to this section need further improvement. Kindly ensure to improve upon this aspect below:
- [ ] The white text is overflowing its container. Kindly improve upon this aspect below by ensuring that the white text remains in its parent grey container with appropriate spacing around it, as can be seen in  the [template design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)

**Your main program section:**
![image](https://user-images.githubusercontent.com/12745474/137292506-a03e4aa7-8681-44bf-83e3-7f36d8a4b442.png)


**the original main program section:**
![image](https://user-images.githubusercontent.com/12745474/137292599-6c9d5c1a-f6a4-4748-ba55-eb435b51e237.png)